lms/djangoapps/bulk_email/tests/test_email.py

```diff
@@ -90,6 +92,7 @@ def tearDown(self):
         """
         patch.stopall()
 
+    @patch('bulk_email.models.html_to_text', Mock(return_value='Mocking CourseEmail.text_message'))
```

Comment:
Please refactor out the testcases that need to be patched into a subclass and patch that in one place.